### ISSUE TITLE
Add settingslogic in Config

### ISSUE THREAD

**Issue by n-rodriguez** [markets/awesome-ruby#860]
Hi there! Can you please add [settingslogic](https://github.com/binarylogic/settingslogic)?

> A simple and straightforward settings solution that uses an ERB enabled YAML file and a singleton design pattern.

It's about 5.5M downloads.

Thank you!

**Comment by markets** [comment]
See https://github.com/settingslogic/settingslogic/issues/15

I think we already have a pretty decent list of "config" gems, with activity and releases these days.

**Comment by n-rodriguez** [comment]
> I think we already have a pretty decent list of "config" gems, with activity and releases these days.

yes and no... [econfig](https://github.com/varvet/econfig) has no doc, no build, and 30k downloads...

**Comment by n-rodriguez** [comment]
The lack of activity on the repo doesn't mean it doesn't work ;) The gem's code is pretty straightforward and well tested. I'm not pushing settingslogic against econfig, but maybe we should replace econfig by settingslogic in regards of the quality requirements.
